cpu/cortexm_common/thread_arch.c

```diff
@@ -366,3 +370,81 @@ void __attribute__((naked)) __attribute__((used)) isr_svc(void) {
     /* {r0-r3,r12,LR,PC,xPSR,s0-s15,FPSCR} are restored automatically on exception return */
     );
 }
+
+#ifdef MODULE_CORTEXM_SVC
+void svc_dispatch(unsigned int *svc_args);
```

Comment:
How will this function be used? shouldn't this be in `cpu.h`?

Reply:
It is called by `isr_svc()`.

Reply:
Sorry I didn't express my question correctly, I mean shouldn't this be private/static in that case? Maybe using `__attribute__((used))`?

Reply:
I've made it static!